tensorboard/webapp/metrics/store/metrics_reducers.ts

```diff
@@ -979,18 +978,22 @@ const reducer = createReducer(
       stepSelectorEnabled: nextStepSelectorEnabled,
     };
   }),
-  on(actions.stepSelectorRangeToggled, (state) => {
-    const nextStepSelectorRangeEnabled = !state.stepSelectorRangeEnabled;
+  on(actions.rangeSelectionToggled, (state) => {
+    const nextRangeSelectionEnabled = !state.rangeSelectionEnabled;
     let nextStepSelectorEnabled = state.stepSelectorEnabled;
 
-    if (nextStepSelectorRangeEnabled) {
-      nextStepSelectorEnabled = nextStepSelectorRangeEnabled;
+    let linkedTimeSelection = state.linkedTimeSelection
+      ? {...state.linkedTimeSelection}
+      : null;
+    if (nextRangeSelectionEnabled) {
+      nextStepSelectorEnabled = nextRangeSelectionEnabled;
     }
 
     return {
       ...state,
+      linkedTimeSelection,
```

Comment:
I do not understand what you are trying to accomplish with by setting this here.

Reply:
Looks like I missed pulling this when I split the step selector portion of out this pr.